<!DOCTYPE html>
<html><head><script src="/livereload.js?mindelay=10&amp;v=2&amp;port=1313&amp;path=livereload" data-no-instant defer></script>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    
    <meta name="description" content="//localhost:1313/">
    <meta name="keywords" content="personal, science, aging">

    <meta property="og:site_name" content="「应可钧」Kejun Albert Ying">
    <meta property="og:title" content="「应可钧」Kejun Albert Ying">
    <meta property="og:description" content="Research">
    <meta property="og:type" content="website">
    <meta property="og:url" content="//localhost:1313/english/research/">
    <meta property="og:image" content="//localhost:1313/">
    <meta name="twitter:card" content="summary">
    <meta name="twitter:site" content="//localhost:1313/english/research/">
    <meta name="twitter:image" content="//localhost:1313/">

    
    <link rel="stylesheet" type="text/css" href="//localhost:1313/css/style.css">
    
    
    
        
        <link rel="stylesheet" href="/custom_style.css" integrity="">
    

    <link rel="stylesheet" href="https://use.typekit.net/jxt2fnp.css"> 
    <link rel="stylesheet" href="https://use.typekit.net/uwd7war.css"> 
    <link href="https://fonts.cdnfonts.com/css/cooper-hewitt" rel="stylesheet">
    
    
        <meta name="description" content="Kejun (Albert) Ying&#39;s personal website | 应可钧的个人网站">
    

    
        
            <link rel="shortcut icon" href="/avatar.jpg">
        
    

    <script defer src="https://kejunying.com/share/script.js" data-website-id="ce4b7b41-6d20-47f0-81b2-e316006cc9f2"></script>
    <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
    
    
    
    <title>research | 「应可钧」kejun albert ying</title>
</head>
<body><div id="nav-border" class="container">
    
    <div class="nav-outer">
        <div class="article-nav-inner">
            
                
                    
                        <a class="internal" hyphens="none" href="/">
                            
                            Home
                        </a>
                    
                
                    
                        <a class="internal" hyphens="none" href="/talks">
                            
                            Talks
                        </a>
                    
                
                    
                        <a class="internal" hyphens="none" href="/research">
                            
                            Research
                        </a>
                    
                
                    
                        <a class="internal" hyphens="none" href="/cv">
                            
                            Cv
                        </a>
                    
                
                    
                        <a class="internal" hyphens="none" href="/about">
                            
                            About
                        </a>
                    
                
                    
                        <a class="internal" hyphens="none" href="/poetry">
                            
                            Poetry
                        </a>
                    
                
            
        </div>
    </div>
    
</div>


<div id="content">

<h2 class="article-title">Research</h2>
<div id="doc">
    <div class="hanging-subtitle">
        
                
        
    </div>
<style type="text/css">
ul li:before {
  content: "";
  margin: 0;
}

ul li p {
  margin-left: 0;
}

ul li p, .sidenote {
  font-size: 80%;
  line-height: 1.4;
}

ul li a {
  padding: 0 2px 0 2px;
}
</style>
<p>Selection of publications. For a full list see my <a href="https://scholar.google.com/citations?user=JS7AdkcAAAAJ&amp;hl=en">Google Scholar profile</a></p>
<h1 id="2024"><strong>2024</strong></h1>
<div class="sidenote" ><div class='altmetric-embed' data-badge-popover='bottom' data-doi='10.1038/s43587-023-00557-0' data-hide-less-than=10 data-badge-type="donut"></div>
<p><img src="https://media.springernature.com/w440/springer-static/cover-hires/journal/43587/4/2?as=webp&q=95" class='img-small'></img></p></div>
<h4 id="causality-enriched-epigenetic-age-uncouples-damage-and-adaptationhttpsdoiorg101038s43587-023-00557-0"><a href="https://doi.org/10.1038/s43587-023-00557-0">Causality-Enriched Epigenetic Age Uncouples Damage and Adaptation</a></h4>
<h5 id="insk-yingins-h-liu-ae-tarkhov-mc-sadler-at-lu-m-moqri--vn-gladyshev"><ins><strong>K Ying</strong>,</ins> H Liu, AE Tarkhov, MC Sadler, AT Lu, M Moqri, &hellip;, VN Gladyshev</h5>
<p><em><strong>Nature Aging</strong> (Featured on the February cover)</em></p>
<p><code>tldr</code>: This work established a novel framework to introduce causal information into epigenetic clocks, resulting in DamAge and AdaptAge—clocks that track detrimental and adaptive methylation changes, respectively.</p>
<br/>
<div class="sidenote" ><div class='altmetric-embed' data-badge-popover='bottom' data-doi='10.1038/s41591-023-02784-9' data-hide-less-than=10 data-badge-type="donut"></div>
</div>
<h4 id="validation-of-biomarkers-of-aginghttpsdoiorg101038s41591-023-02784-9"><a href="https://doi.org/10.1038/s41591-023-02784-9">Validation of Biomarkers of Aging</a></h4>
<h5 id="m-moqri-c-herzog-jr-poganik-insk-yingins-jn-justice-dw-belsky--l-ferrucci">M Moqri, C Herzog, JR Poganik, <ins><strong>K Ying</strong></ins>, JN Justice, DW Belsky, &hellip;, L Ferrucci</h5>
<p><em><strong>Nature Medicine</strong></em></p>
<p><code>tldr:</code> Current efforts to evaluate the predictive validity of omic biomarkers of aging in population studies are reviewed, challenges in comparability and generalizability are discussed, and recommendations to facilitate future validation of biomarkers of aging are provided</p>
<div class="sidenote" ><div class='altmetric-embed' data-badge-popover='bottom' data-doi='10.1038/s43587-023-00555-2' data-hide-less-than=10 data-badge-type="donut"></div>
</div>
<h4 id="time-seq-reduces-time-and-cost-of-dna-methylation-measurement-for-epigenetic-clock-constructionhttpsdoiorg101038s43587-023-00555-2"><a href="https://doi.org/10.1038/s43587-023-00555-2">TIME-seq Reduces Time and Cost of DNA Methylation Measurement for Epigenetic Clock Construction</a></h4>
<h5 id="pt-griffin-ae-kane-a-trapp-j-li-m-arnold-jr-poganik-rj-conway-ms-mcnamara-mv-meer-n-hoffman-ja-amorim-x-tian-mr-macarthur-sj-mitchell-al-mueller-c-carmody-dl-vera-c-kerepesi-insk-yingins--da-sinclair">PT Griffin, AE Kane, A Trapp, J Li, M Arnold, JR Poganik, RJ Conway, MS McNamara, MV Meer, N Hoffman, JA Amorim, X Tian, MR MacArthur, SJ Mitchell, AL Mueller, C Carmody, DL Vera, C Kerepesi, <ins><strong>K Ying</strong></ins>, &hellip;, DA Sinclair</h5>
<p><em><strong>Nature Aging</strong></em></p>
<h1 id="2023"><strong>2023</strong></h1>
<div class="sidenote" ><div class='altmetric-embed' data-badge-popover='bottom' data-doi='10.1016/j.molcel.2023.08.014' data-hide-less-than=10 data-badge-type="donut"></div>
</div>
<h4 id="18s-rrna-methyltransferases-dimt1-and-bud23-drive-intergenerational-hormesishttpsdoiorg101016jmolcel202308014"><a href="https://doi.org/10.1016/j.molcel.2023.08.014">18S rRNA Methyltransferases DIMT1 and BUD23 Drive Intergenerational Hormesis</a></h4>
<h5 id="n-liberman-mh-rothi-mv-gerashchenko-c-zorbas-k-boulias-fg-macwhinnie-insak-yingins--el-greer">N Liberman, MH Rothi, MV Gerashchenko, C Zorbas, K Boulias, FG MacWhinnie, <ins><strong>AK Ying</strong></ins>, &hellip;, EL Greer</h5>
<p><em><strong>Molecular Cell</strong></em></p>
<div class="sidenote" ><div class='altmetric-embed' data-badge-popover='bottom' data-doi='10.1038/s42255-023-00815-w' data-hide-less-than=10 data-badge-type="donut"></div>
</div>
<h4 id="acarbose-suppresses-symptoms-of-mitochondrial-disease-in-a-mouse-model-of-leigh-syndromehttpsdoiorg101038s42255-023-00815-w"><a href="https://doi.org/10.1038/s42255-023-00815-w">Acarbose Suppresses Symptoms of Mitochondrial Disease in a Mouse Model of Leigh Syndrome</a></h4>
<h5 id="a-bitto-as-grillo-tk-ito-ib-stanaway-bmg-nguyen-insk-yingins--m-kaeberlein">A Bitto, AS Grillo, TK Ito, IB Stanaway, BMG Nguyen, <ins><strong>K Ying</strong></ins>, &hellip;, M Kaeberlein</h5>
<p><em><strong>Nature Metabolism</strong></em></p>
<div class="sidenote" ><div class='altmetric-embed' data-badge-popover='bottom' data-doi='10.1101/2023.02.28.530532' data-hide-less-than=10 data-badge-type="donut"></div>
<p><img src="https://lh4.googleusercontent.com/_t5umxBnHGExx7Rj3I4ZJgCcib99i4I7yN8ViHtwJmZmjavmDb_i605h76mVszX3SILrBag25BFfJ39vrVOaBDI=w16383" class='img-small'></img></p></div>
<h4 id="clockbase-a-comprehensive-platform-for-biological-age-profiling-in-human-and-mousehttpsdoiorg10110120230228530532"><a href="https://doi.org/10.1101/2023.02.28.530532">ClockBase: A Comprehensive Platform for Biological Age Profiling in Human and Mouse</a></h4>
<h5 id="insk-yingins-a-tyshkovskiy-a-trapp-h-liu-m-moqri-c-kerepesi-vn-gladyshev"><ins><strong>K Ying</strong></ins>, A Tyshkovskiy, A Trapp, H Liu, M Moqri, C Kerepesi, VN Gladyshev</h5>
<p><em><strong>bioRxiv</strong></em></p>
<p><code>tldr:</code> ClockBase is a comprehensive platform offering biological age estimates from over 2,000 DNA methylation datasets and nearly 200,000 samples using various aging clock models. It includes an online interface for data analysis and visualization. The platform facilitated the identification of potential anti-aging drugs, thereby aiding in the discovery of interventions for longevity and conditions that accelerate aging. The website is <a href="clockbase.org">clockbase.org</a>.</p>
<div class="sidenote" ><div class='altmetric-embed' data-badge-popover='bottom' data-doi='10.1101/2023.12.02.569722' data-hide-less-than=10 data-badge-type="donut"></div>
</div>
<h4 id="biolearn-an-open-source-library-for-biomarkers-of-aginghttpsdoiorg10110120231202569722"><a href="https://doi.org/10.1101/2023.12.02.569722">Biolearn, an Open-Source Library for Biomarkers of Aging</a></h4>
<h5 id="insk-yingins-s-paulson-m-perez-guevara-m-emamifar-m-casas-martinez-d-kwon-jr-poganik-m-moqri-vn-gladyshev"><ins><strong>K Ying</strong></ins>, S Paulson, M Perez-Guevara, M Emamifar, M Casas Martinez, D Kwon, JR Poganik, M Moqri, VN Gladyshev</h5>
<p><em><strong>bioRxiv</strong></em></p>
<p><code>tldr:</code> By furnishing a community-driven platform, Biolearn significantly augments the development, assessment, and validation trajectories of aging biomarkers, paving the way toward more rigorous clinical validation and, ultimately, application in clinical trials targeting healthy longevity.</p>
<h1 id="2022"><strong>2022</strong></h1>
<div class="sidenote" ><div class='altmetric-embed' data-badge-popover='bottom' data-doi='10.1161/CIRCULATIONAHA.121.057888' data-hide-less-than=10 data-badge-type="donut"></div>
</div>
<h4 id="genetic-landscape-of-the-ace2-coronavirus-receptorhttpsdoiorg101161circulationaha121057888"><a href="https://doi.org/10.1161/circulationaha.121.057888">Genetic Landscape of the ACE2 Coronavirus Receptor</a></h4>
<h5 id="z-yang-e-macdonald-dunlop-j-chen-r-zhai-t-li-a-richmond--insk-yingins--x-shen">Z Yang, E Macdonald-Dunlop, J Chen, R Zhai, T Li, A Richmond, &hellip;, <ins><strong>K Ying</strong></ins>, &hellip;, X Shen</h5>
<p><em><strong>Circulation</strong></em></p>
<div class="sidenote" ><div class='altmetric-embed' data-badge-popover='bottom' data-doi='10.1101/2022.08.02.502559' data-hide-less-than=10 data-badge-type="donut"></div>
</div>
<h4 id="epigenetic-profiling-and-incidence-of-disrupted-development-point-to-gastrulation-as-aging-ground-zero-in-xenopus-laevishttpsdoiorg10110120220802502559"><a href="https://doi.org/10.1101/2022.08.02.502559">Epigenetic Profiling and Incidence of Disrupted Development Point to Gastrulation as Aging Ground Zero in Xenopus Laevis</a></h4>
<h5 id="b-zhang-ae-tarkhov-w-ratzan-insk-yingins-m-moqri-jr-poganik--vn-gladyshev">B Zhang, AE Tarkhov, W Ratzan, <ins><strong>K Ying</strong></ins>, M Moqri, JR Poganik, &hellip;, VN Gladyshev</h5>
<p><em><strong>bioRxiv</strong></em></p>
<div class="sidenote" ><div class='altmetric-embed' data-badge-popover='bottom' data-doi='10.1101/2022.09.26.509592' data-hide-less-than=10 data-badge-type="donut"></div>
</div>
<h4 id="nature-of-epigenetic-aging-from-a-single-cell-perspectivehttpsdoiorg10110120220926509592"><a href="https://doi.org/10.1101/2022.09.26.509592">Nature of Epigenetic Aging from a Single-Cell Perspective</a></h4>
<h5 id="ae-tarkhov-t-lindstrom-vautrin-s-zhang-insk-yingins-m-moqri-b-zhang-vn-gladyshev">AE Tarkhov, T Lindstrom-Vautrin, S Zhang, <ins><strong>K Ying</strong></ins>, M Moqri, B Zhang, VN Gladyshev</h5>
<p><em><strong>bioRxiv</strong></em></p>
<h1 id="2021"><strong>2021</strong></h1>
<div class="sidenote" ><div class='altmetric-embed' data-badge-popover='bottom' data-doi='10.1038/s43856-021-00033-z' data-hide-less-than=10 data-badge-type="donut"></div>
</div>
<h4 id="genetic-and-phenotypic-analysis-of-the-causal-relationship-between-aging-and-covid-19httpsdoiorg101038s43856-021-00033-z"><a href="https://doi.org/10.1038/s43856-021-00033-z">Genetic and Phenotypic Analysis of the Causal Relationship between Aging and COVID-19</a></h4>
<h5 id="insk-yingins-r-zhai-tv-pyrkov-av-shindyapina-m-mariotti-po-fedichev-x-shen-vn-gladyshev"><ins><strong>K Ying</strong></ins>, R Zhai, TV Pyrkov, AV Shindyapina, M Mariotti, PO Fedichev, X Shen, VN Gladyshev</h5>
<p><em><strong>Communications Medicine</strong></em></p>
<p><code>tldr:</code> The genetic variation associated with longer lifespan is associated with reduced risk of COVID-19 infection and hospitalization, and some of the key cells and molecules involved in aging-related CO VID-19 risk are identified.</p>
<div class="sidenote" ><div class='altmetric-embed' data-badge-popover='bottom' data-doi='10.1038/s41467-021-23124-w' data-hide-less-than=10 data-badge-type="donut"></div>
</div>
<h4 id="total-genetic-contribution-assessment-across-the-human-genomehttpsdoiorg101038s41467-021-23124-w"><a href="https://doi.org/10.1038/s41467-021-23124-w">Total Genetic Contribution Assessment across the Human Genome</a></h4>
<h5 id="t-li-z-ning-z-yang-r-zhai-c-zheng-w-xu-y-wang-insk-yingins-y-chen-x-shen">T Li, Z Ning, Z Yang, R Zhai, C Zheng, W Xu, Y Wang, <ins><strong>K Ying</strong></ins>, Y Chen, X Shen</h5>
<p><em><strong>Nature Communications</strong></em></p>
<h1 id="2019"><strong>2019</strong></h1>
<div class="sidenote" ><div class='altmetric-embed' data-badge-popover='bottom' data-doi='10.1007/s13238-018-0593-9' data-hide-less-than=10 data-badge-type="donut"></div>
</div>
<h4 id="phosphorylation-of-plin3-by-ampk-promotes-dispersion-of-lipid-droplets-during-starvationhttpsdoiorg101007s13238-018-0593-9"><a href="https://doi.org/10.1007/s13238-018-0593-9">Phosphorylation of PLIN3 by AMPK Promotes Dispersion of Lipid Droplets during Starvation</a></h4>
<h5 id="j-zhu-m-xu-y-liu-l-zhuang-insk-yingins-f-liu-d-liu-w-ma-z-songyang">J Zhu, M Xu, Y Liu, L Zhuang, <ins><strong>K Ying</strong></ins>, F Liu, D Liu, W Ma, Z Songyang</h5>
<p><em><strong>Protein &amp; Cell</strong></em></p>

</div>

        </div>
<link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.5.1/styles/foundation.min.css">
<script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/11.5.1/highlight.min.js"></script>
<script>
  hljs.highlightAll();
</script>


<script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
<script>
  MathJax = {
      tex: {
        inlineMath: [ ['$','$'] ]
      }
  };
</script>
<script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js"></script>


<script src="https://unpkg.com/feather-icons"></script>
<script>
  feather.replace()
</script>

<div id="top-stripe"></div>
</body>
</html>
